packages/mds-compliance/api.ts

```diff
@@ -31,7 +31,7 @@ import {
   AuthorizationError
 } from '@mds-core/mds-utils'
 import { Geography, Device, UUID, VehicleEvent } from '@mds-core/mds-types'
-import { TEST1_PROVIDER_ID, TEST2_PROVIDER_ID, BLUE_SYSTEMS_PROVIDER_ID, providerName } from '@mds-core/mds-providers'
+import { providerName } from '@mds-core/mds-providers'
```

Comment:
Curious what `providerName` is used for. Overall, we should remove the use of this whenever possible since it's based on a hard coded list of provider id values.

Reply:
It's there to convert provider_ids to a human-readable name if we have one. Agree re: removing hard-coded provider_ids, but a little out of scope for this.